content/en/platform/corda/4.6/open-source/api-flows.md

```diff
@@ -1831,7 +1831,7 @@ another flow to take control and run.
 Implementations of `FlowExternalAsyncOperation` must return a `CompletableFuture`. How this future is created is up to the developer.
 It is recommended to use `CompletableFuture.supplyAsync` and supply an executor to run the future on. Other libraries can be used to
 generate futures, as long as a `CompletableFuture` is returned out of `FlowExternalAsyncOperation`. An example of creating a future
-using [Guava’s ListenableFuture](#api-flows-guava-future-conversion) is given in a following section.
+using [Guava’s ListenableFuture](#creating-completablefutures-from-guavas-listenablefutures) is given in a following section.
 
```

Comment:
There is a broken link in this file here: 
![image](https://user-images.githubusercontent.com/69467840/137772131-118e0469-a17d-4a54-8725-47725b96d636.png)

Reply:
Looks like an .md that needs to be an .html

Reply:
Another here: 
![image](https://user-images.githubusercontent.com/69467840/137772297-eab99e00-0beb-4311-8a14-fc1f8c5f4698.png)

Reply:
Good catch, thanks!